lib/newsComponent/viewModel/news_viewmodel.dart

```diff
@@ -1,39 +1,29 @@
 import 'package:campus_flutter/base/networking/protocols/view_model.dart';
-import 'package:campus_flutter/newsComponent/model/news.dart';
-import 'package:campus_flutter/newsComponent/model/news_source.dart';
 import 'package:campus_flutter/newsComponent/service/news_service.dart';
-import 'package:collection/collection.dart';
 import 'package:rxdart/rxdart.dart';
 
+import '../../base/networking/apis/tumdev/campus_backend.pb.dart';
+
 class NewsViewModel implements ViewModel {
-  BehaviorSubject<List<NewsSource>?> newsSources = BehaviorSubject.seeded(null);
+  BehaviorSubject<List<News>?> news = BehaviorSubject.seeded(null);
 
   final BehaviorSubject<DateTime?> lastFetched = BehaviorSubject.seeded(null);
 
   @override
   Future fetch(bool forcedRefresh) async {
-    NewsService.fetchNews(false)
-        .then((value) {
+    NewsService.fetchNews(forcedRefresh).then((value) {
       lastFetched.add(value.$1);
-      newsSources.add(value.$2);
-    }, onError: (error) => newsSources.addError(error));
+      news.add(value.$2);
+    }, onError: (error) => news.addError(error));
   }
 
-  List<(String?, News)> latestFiveNews() {
-    if (newsSources.value != null) {
-     final news = newsSources.value!.expand((element) => element.news).toList();
-
-      news.removeWhere((element) => (element.source == "2"));
-
-      news.sort((news1, news2) => news2.created.compareTo(news1.created));
-
-      List<News> fiveNews = news.sublist(0, 5);
-
-      return fiveNews
-          .map((e) => (newsSources.value!.firstWhereOrNull((element) => e.source == element.id.toString())?.title, e))
-          .toList();
-    } else {
+  List<News> latestFiveNews() {
+    if (news.value == null) {
       return [];
     }
+    final newsItems = news.value!.toList();
+    newsItems.removeWhere((element) => (element.source == "2"));
```

Comment:
This is bug-per-bug the same logic, but I am pretty sure that we should not 
- sort on `created` (which is when it was created in our backend), instead on `date` (which is the time the news was released)
- exclude source 2 without any comment what this is

Reply:
> * sort on `created` (which is when it was created in our backend), instead on `date` (which is the time the news was released)

Makes sense, we can change that.

> * exclude source 2 without any comment what this is

I copied that from Campus-iOS, doesn't seem to be relevant since [https://app.tum.app/api/news/sources](url) doesn't list that as a source anymore. It also seems to me that TUM-News (source 1) is not working properly.